tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "noEmit": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "strict": false,
+    "strict": true,
```

Comment:
If it still passes the tests it's a great change.

Reply:
It sure does!